.changeset/spicy-suits-explode.md

```diff
@@ -0,0 +1,23 @@
+---
+"astro": minor
+---
+
+The Astro Actions API introduced behind a flag in [v4.8.0](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md#480) is no longer experimental and is available for general use.
+
+Astro Actions allow you to define and call backend functions with type-safety. Actions perform data fetching, JSON parsing, and input validation for you. This can greatly reduce the amount of boilerplate needed compared to using an API endpoint.
```

Comment:
No notes on the included text! Just giving you permission in case you wanted to expand here a bit with an example, maybe like how the experimental changeset would have looked (updated and correct of course) or copying from the deleted experimental config reference content below.

I will not un-approve if this gets longer! At your discretion, @bholmesdev !

Reply:
Haha, appreciate that. I'll copy a snippet from the 4.15 release blog post